8_child_illness.do

```diff
@@ -387,6 +404,9 @@ c_illtreat
 		}
 		if inlist(country_name,"CostaRica2018") {
 			global ca21 "ca21a ca21b ca21c ca21e ca21i ca21j ca21m"
+		}
+		if inlist(country_name,"Tonga2019") {
```

Comment:
Let's walk through logics here just in case in chat.

Reply:
checked